nss-cohort-36/bangazon-client-virtual-magicians-1
# Search Bar

# Description
Fuzzy search from the navbar

## Type of change

Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
```shell
git checkout master
git fetch --all
git checkout gc-search-for-products
npm start
```

- Make sure that you're authenticated 
- Try to search for something from the search bar and expect the applicationview part to change to the searchresults components showing the relevant results.
- Clear the search bar and expect to be redirected to Home view



# Checklist:
